pkg/storegateway/chunk_bytes_pool.go

```diff
@@ -30,6 +32,10 @@ func newChunkBytesPool(minBucketSize, maxBucketSize int, maxChunkPoolBytes uint6
 			Name: "cortex_bucket_store_chunk_pool_returned_bytes_total",
 			Help: "Total bytes returned by the chunk bytes pool.",
 		}),
+		poolByteStats: promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Name: "cortex_bucket_store_chunk_pool_bytes_total",
```

Comment:
Umm, not sure if we can find a better name... Is it too general?

Reply:
u have a better one :P